File: KP/revision/W2/JAVA/TablicaBoolTest.java
public class TablicaBoolTest {
    public static void main (String Args[]) {
        TablicaBool dziennik = new TablicaBool(25);
        boolean uczen;

        dziennik.Obecny(9);
        dziennik.Nieobecny(15);

        try {
            uczen = dziennik.CzyObecny(9);
            if(uczen) {
                System.out.println("UCZEN O INDEKSIE 9 JEST OBECNY");
            }

            uczen = dziennik.CzyObecny(15);
            if(uczen) {
                System.out.println("UCZEN O INDEKSIE 15 JEST OBECNY");
            }
        } catch(MojWyjatek ex) {
            System.out.println("Podano bledny indeks ucznia");
        }

        try {
            uczen = dziennik.CzyObecny(89);
        } catch(MojWyjatek ex) {
            System.out.println("Podano bledny indeks ucznia");
        }
    }
}

File: KP/revision/W2/CPP/TablicaBoolTest.cpp
#include <iostream>
#include "TablicaBool.hpp"

using namespace std;

int main(int args, char* Arg[]) {
    TablicaBool *dziennik = new TablicaBool(25);
    bool uczen;

    dziennik->Obecny(9);
    dziennik->Nieobecny(13);

    try {
        uczen = dziennik->CzyObecny(9);
        if(uczen)
            cout << "UCZEN 9 JEST OBECNY" << endl;

        uczen = dziennik->CzyObecny(13);
        if(uczen)
            cout << "UCZEN 13 JEST OBECNY" << endl;
    } catch(string e) {
        cout << e << endl;
    }

    try {
        uczen = dziennik->CzyObecny(34);
    } catch(string e) {
        cout << e << endl;
    }

    delete dziennik;
}
